File: Praktikum Modul 4/SOAL-4/BAHASA-C/PRAK404-2410817110014-RIZKI_ADHITIYA_MAULANA.c
#include <stdio.h>

int main ()
{
    int pilihan;
    float nilai_pertama, nilai_kedua, hasil;

    while (1)
    {
    printf("Pilih program\n");
    printf("1. Penjumlahan\n");
    printf("2. Pengurangan\n");
    printf("3. Perkalian\n");
    printf("4. Pembagian\n");
    printf("5. Exit\n");

    printf("Masukkan Pilihan : ");
    scanf("%d", &pilihan);

        if (pilihan == 1)
        {
            printf("Masukkan nilai pertama : ");
            scanf("%f", &nilai_pertama);
            printf("Masukkan nilai kedua : ");
            scanf("%f", &nilai_kedua);
            hasil = nilai_pertama + nilai_kedua;
            printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n\n", nilai_pertama, nilai_kedua, hasil);
            continue;
        }

        else if (pilihan == 2)
        {
            printf("Masukkan nilai pertama : ");
            scanf("%f", &nilai_pertama);
            printf("Masukkan nilai kedua : ");
            scanf("%f", &nilai_kedua);
            hasil = nilai_pertama - nilai_kedua;
            printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n\n", nilai_pertama, nilai_kedua, hasil);
            continue;
        }

        else if (pilihan == 3)
        {
            printf("Masukkan nilai pertama : ");
            scanf("%f", &nilai_pertama);
            printf("Masukkan nilai kedua : ");
            scanf("%f", &nilai_kedua);
            hasil = nilai_pertama * nilai_kedua;
            printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n\n", nilai_pertama, nilai_kedua, hasil);
            continue;
        }

        else if (pilihan == 4)
        {
            printf("Masukkan nilai pertama : ");
            scanf("%f", &nilai_pertama);
            printf("Masukkan nilai kedua : ");
            scanf("%f", &nilai_kedua);
            hasil = nilai_pertama / nilai_kedua;
            printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f\n\n", nilai_pertama, nilai_kedua, hasil);
            continue;
        }

        else if (pilihan == 5)
        {
            printf("Terimakasih, telah menggunakan kalkulator RIZKI ADHITIYA MAULANA");
            break;
        }
    
        else
        {
        printf("Input anda salah, silahkan coba lagi\n\n");
        continue;
        }  
    }
    return 0;
}

File: Praktikum Modul 4/SOAL-4/BAHASA-PY/PRAK404-2410817110014-RIZKI_ADHITIYA_MAULANA.py
while True:
    print("Pilih program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")

    pilihan = int(input("Masukkan pilihan : "))
    
    if pilihan == 1:
        nilai_pertama = float(input("Masukkan nilai pertama : "))
        nilai_kedua = float(input("Masukkan nilai kedua : "))
        hasil = nilai_pertama + nilai_kedua
        print(f"Hasil perjumlahan antara {nilai_pertama:.2f} dengan {nilai_kedua:.2f} adalah {hasil:.2f}")
        print("")

    elif pilihan == 2:
        nilai_pertama = float(input("Masukkan nilai pertama : "))
        nilai_kedua = float(input("Masukkan nilai kedua : "))
        hasil = nilai_pertama - nilai_kedua
        print(f"Hasil pengurangan antara {nilai_pertama:.2f} dengan {nilai_kedua:.2f} adalah {hasil:.2f}")
        print("")

    elif pilihan == 3:
        nilai_pertama = float(input("Masukkan nilai pertama : "))
        nilai_kedua = float(input("Masukkan nilai kedua : "))
        hasil = nilai_pertama * nilai_kedua
        print(f"Hasil perkalian antara {nilai_pertama:.2f} dengan {nilai_kedua:.2f} adalah {hasil:.2f}")
        print("")

    elif pilihan == 4:
        nilai_pertama = float(input("Masukkan nilai pertama : "))
        nilai_kedua = float(input("Masukkan nilai kedua : "))
        hasil = nilai_pertama / nilai_kedua
        print(f"Hasil pembagian antara {nilai_pertama:.2f} dengan {nilai_kedua:.2f} adalah {hasil:.2f}")
        print("")
    
    elif pilihan == 5:
        print("Terimakasih, telah menggunakan kalkulator RIZKI ADHITIYA MAULANA")
        break

    else:
        print("Input anda salah, silahkan coba lagi")
        print("")
        continue